rest/model/task.go

```diff
@@ -76,6 +76,11 @@ type APITask struct {
 	AMI                     *string             `json:"ami"`
 	MustHaveResults         bool                `json:"must_have_test_results"`
 	BaseTask                APIBaseTaskInfo     `json:"base_task"`
+
+	// These fields are only for cached tasks (tasks displaying info from a previous task)
+	IsCachedResult bool `json:"is_cached"`
+	FromExecution  int  `json:"from_execution"`
```

Comment:
do we want  to differentiate between 0 and nil?

Reply:
I changed to be an int pointer with omitempty so it doesn't show up with clients using the api routes (this is in APITask), but it will be accessible via spruce so data displaying it was cached is there.